lib/css-converter.js

```diff
@@ -0,0 +1,319 @@
+import { CssSelectorParser } from 'css-selector-parser';
+import _ from 'lodash';
```

Comment:
Can't we just do this 

```suggestion
import { escapeRegExp } from 'lodash';
```

and change the `_ escapeRegExp` into `escapeRegExp` like we normally do with imports?